pkg/networkmanager/team.jsx

```diff
@@ -159,19 +159,19 @@
                     <MemberInterfaceChoices idPrefix={idPrefix} memberChoices={memberChoices} setMemberChoices={setMemberChoices} model={model} group={connection} />
                 </FormGroup>
                 <FormGroup fieldId={idPrefix + "-runner-select"} label={_("Runner")}>
-                    <FormSelect id={idPrefix + "-runner-select"} onChange={setRunner}
+                    <FormSelect id={idPrefix + "-runner-select"} onChange={(_, val) => setRunner(val)}
                                 value={runner}>
                         {team_runner_choices.map(choice => <FormSelectOption value={choice.choice} label={choice.title} key={choice.choice} />)}
                     </FormSelect>
                 </FormGroup>
                 {(runner == "loadbalance" || runner == "lacp") && <FormGroup fieldId={idPrefix + "-balancer-select"} label={_("Balancer")}>
-                    <FormSelect id={idPrefix + "-balancer-select"} onChange={setBalancer}
+                    <FormSelect id={idPrefix + "-balancer-select"} onChange={(_, val) => setBalancer(val)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18739-20230503-161849-e27cc6bb-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)